### ISSUE TITLE
Use LibBS to collect artifacts from decompilers

### ISSUE THREAD

**Issue by mahaloz** [mahaloz/decomp2dbg#84]
## Overview
[LibBS](https://github.com/binsync/libbs), an abstracted library for working with decompilers from the BinSync team, implements much of what we do in XMLRPC servers for the many decompilers. It also reimplements all the Java Ghidra code we have into Python. 

This change would remove all code from the [decompilers](https://github.com/mahaloz/decomp2dbg/tree/main/decompilers) folder, since it is contained in LibBS

## Benefits
- Reduces the overlapping code I've written for BinSync and DAILA contained in d2d
- We gain bug fixes from fixes to BinSync
- We get one step closer to https://github.com/mahaloz/decomp2dbg/issues/36, since we will now have struct exporting supported in all decompilers

## Disadvantages
- Debugging for new people to the project gets harder since now they need to clone LibBS to debug problems that occur inside the decompiler servers accurately (harder debugging)
- Installing this tool gets a _little_ harder, since now `decomp2dbg`, the Python package, will need to be installed **in the same Python env as the decompiler**. For Ghidra, this is trivial since one does not exist; they install it into the Python found in the path. For Binja, it's a possible pain. IDA is ok. 

---

This PR is stalled by this issue from LibBS:
https://github.com/binsync/libbs/issues/16

Once it is done, this can be worked on. 
Since this is open-source, if the community is reading this issue now, I highly recommend you react to the issue with a 👍  or a 👎  depending on how you feel about the PR. I.e., if you want this, 👍. Otherwise, 👎.

**Comment by k4lizen** [comment]
Seems the libbs PR was merged!

**Comment by mahaloz** [comment]
Good point! Now I just need to find some time 😢. Struct support would be amazing, but caching would be EVEN BETTER.
